app.js

```diff
@@ -0,0 +1,54 @@
+var cookieParser = require("cookie-parser");
```

Comment:
Please use `const` for imports rather than `var`.

Reply:
it is a part of boilerplate, Bdw i will change it.